go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/IBM/idemix
 go 1.19
```

Comment:
what about moving to 1.21. Go 1.19 is deprecated

Reply:
good point - yes. Given that I have one more functional PR on top of this one, would it be okay to merge this, the next, and then upgrade to go 1.21?

Reply:
sure 👍